src/index.js

```diff
@@ -63,7 +63,10 @@ function createMacro(macro, options = {}) {
 }
 
 function nodeResolvePath(source, basedir) {
-  return resolve.sync(source, {basedir})
+  return resolve.sync(source, {
+    basedir,
+    paths: [p.resolve(__dirname, '../../')]
```

Comment:
Should we include `process.cwd()` in this as well?

Reply:
No, because it is a fallback only. If it cannot resolve relatively to basedir using node's built in resolve mechanism, then it tries with the provided paths as well (in the fallback case there is no travesing up to the parent, that's why `../../` is necessary).

I was using with process.cwd() included tho, but removing it didn't cause any issues. That case can only happen if plugin is installed outside of cwd and basedir is also not with-in cwd, which is highly unlikely imo. Should I add it just for safety?

Reply:
My use case is that I have my devtools globally installed, so at project level I basically don't have any node_modules folder. This means babel-plugin-macros also being installed globally.

Reply:
That's pretty odd. But as this doesn't break anything I'm fine with it.